app/modules/intelligence/agents/chat_agents/multi_agent/utils/multimodal_utils.py

```
@@ -0,0 +1,118 @@
+
+            base64_data = image_data["base64"]
+            if not isinstance(base64_data, str) or not base64_data:
+                logger.error(f"Invalid base64 data for image {attachment_id}")
+                continue
+
+            # Validate base64 format
+            try:
+                import base64
+
+                base64.b64decode(base64_data)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)